src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -3049,8 +3076,15 @@ protected virtual void ProcessMethod (MethodDefinition method, in DependencyInfo
 				}
 			}
 
+			// Mark overrides except for static interface methods
```

Comment:
Don't we want to mark the MethodImpl for these calls if and only if there's a constrained call to the target method?

Reply:
Yes, but here we are marking the methods themselves, not the MethodImpl. The static abstract interface methods themselves are marked elsewhere when the linker finds constrained calls to them. At the moment there's still no real "marking" of the `MethodImpl`, it just checks if the overridden method is marked, and if it's not, it removes the MethodImpl / Override. 

We could separate marking the Overrides list from marking the Overridden methods themselves, but I feel like that would lead to more (issues where the Override / MethodImpl is incorrectly marked and the override method is correctly removed) than (issues where the Override / MethodImpl is correctly marked and the override method is incorrectly removed).

Reply:
Agreed, I like handling this in SweepStep and I like @vitek-karas's reasoning here.